manifests/user.pp

```diff
@@ -36,9 +39,10 @@
   Enum['present', 'absent', 'role'] $ensure              = 'present',
   String                            $username            = $title,
   String                            $group               = $title,
-  String                            $home                = "/home/${title}",
+  Stdlib::AbsolutePath              $home                = "/home/${title}",
   Optional[String]                  $comment             = undef,
   Optional[String]                  $password            = undef,
+  Stdlib::AbsolutePath              $shell               = '/bin/bash',
```

Comment:
I think this should be `undef` by default. However, it is then harder to change.